bitbots_ros_control/src/node.cpp

```diff
@@ -30,7 +30,6 @@ int main (int argc, char *argv[]){
   ros::Rate rate(pnh.param("control_loop_hz", 200));
 
   while (ros::ok()){
-    bool read_sucessfull = hw.read();
```

Comment:
why is this deleted, we still need to read the hardware, we just don't care anymore if it was successful